### PR TITLE
Update fail-to-mount-azure-file-share.md

### PR DESCRIPTION
**Pull request by hanagapal**
Added Cause 5 :
 The below is documentation improvement bug to avoid support cases and CRIs 
Bug 25433172: unable to mount share with custom security settings(Disabled AES128 GCM)

# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 1]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
 Enable the AES-128-GCM algorithm by using the **Maximum compatibility** profile or a **Custom** profile that enables AES-128-GCM. For more information, see [Azure Files Security Settings](/azure/storage/files/files-smb-protocol?tabs=azure-portal#smb-security-settings).
 
+### Cause 5: Minimum encryption requirement for a storage account not met.
+#### Solution AES128-GCM should be enabled for all storage accounts in order to successfully mount or access a file share.
+**Note:**
+For customers who want to use AES 256 GCM encryption only which is Maximum security (SMB3.1.1) should follow the following:
+**Linux:**
+**The following script would check if the client supports AES256 GCM and enforce it only if it is supported:** 
+
+              cifsConfPath="/etc/modprobe.d/cifs.conf"
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:413]:
```suggestion
```bash
cifsConfPath="/etc/modprobe.d/cifs.conf" 
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 2]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
 
+### Cause 5: Minimum encryption requirement for a storage account not met.
+#### Solution AES128-GCM should be enabled for all storage accounts in order to successfully mount or access a file share.
+**Note:**
+For customers who want to use AES 256 GCM encryption only which is Maximum security (SMB3.1.1) should follow the following:
+**Linux:**
+**The following script would check if the client supports AES256 GCM and enforce it only if it is supported:** 
+
+              cifsConfPath="/etc/modprobe.d/cifs.conf" 
+              echo "`date` before change ${cifsConfPath}:"
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:414]:
```suggestion
echo "`date` before change ${cifsConfPath}:"
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 3]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              cat ${cifsConfPath}
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:415]:
```suggestion
cat ${cifsConfPath} 
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 4]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+#### Solution AES128-GCM should be enabled for all storage accounts in order to successfully mount or access a file share.
+**Note:**
+For customers who want to use AES 256 GCM encryption only which is Maximum security (SMB3.1.1) should follow the following:
+**Linux:**
+**The following script would check if the client supports AES256 GCM and enforce it only if it is supported:** 
+
+              cifsConfPath="/etc/modprobe.d/cifs.conf" 
+              echo "`date` before change ${cifsConfPath}:"
+              cat ${cifsConfPath} 
+              if !(( grep require_gcm_256 ${cifsConfPath} ))
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:416]:
```suggestion
if !(( grep require_gcm_256 ${cifsConfPath} )) 
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 5]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+**Note:**
+For customers who want to use AES 256 GCM encryption only which is Maximum security (SMB3.1.1) should follow the following:
+**Linux:**
+**The following script would check if the client supports AES256 GCM and enforce it only if it is supported:** 
+
+              cifsConfPath="/etc/modprobe.d/cifs.conf" 
+              echo "`date` before change ${cifsConfPath}:"
+              cat ${cifsConfPath} 
+              if !(( grep require_gcm_256 ${cifsConfPath} )) 
+              then
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:417]:
```suggestion
then
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 6]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              modprobe cifs
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:418]:
```suggestion
modprobe cifs
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 7]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              echo 1 > /sys/module/cifs/parameters/require_gcm_256
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:419]:
```suggestion
echo 1 > /sys/module/cifs/parameters/require_gcm_256
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 8]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              echo "options cifs require_gcm_256=1" > ${cifsConfPath}
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:420]:
```suggestion
echo "options cifs require_gcm_256=1" > ${cifsConfPath}
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 9]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              echo "`date` after changing ${cifsConfPath}:"
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:421]:
```suggestion
echo "`date` after changing ${cifsConfPath}:"
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 10]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              cat ${cifsConfPath}
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:422]:
```suggestion
cat ${cifsConfPath}
```

File: support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md [hunk 11]
```diff
@@ -403,6 +403,35 @@ AKS versions 1.25 and later versions are based on Ubuntu 22.04, which uses the L
...
+              fi
```
Comment (khdownie) [inline support/azure/azure-kubernetes/fail-to-mount-azure-file-share.md:423]:
```suggestion
fi
```
```

**Review by khdownie — CHANGES_REQUESTED** [review]

**Comment by AmandaAZ** [comment]
@hanagapal @khdownie I see the same changes are submitted via the private [pull request #5101](https://github.com/MicrosoftDocs/SupportArticles-docs-pr/pull/5101), so I closed this PR. Let me know if any problems.

**Pull request closed without merging**